VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        public static string ChangePassword = "PassWord917231";
```

Comment:
[CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.